docs/user-guide/index.md

```diff
@@ -57,6 +57,10 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## WSL (Windows Subsystem for Linux) Users
+
+When selecting connectors, be sure to select `jsdom`, as the `chrome` one can not be used under WSL.
```

Comment:
The [build failed](https://travis-ci.org/sonarwhal/sonar/builds/276602087). This line should be split into multiple lines so to keep the length of the lines to under 80 characters.

Reply:
Do I need to re-edit the file to add a line break then?

Reply:
Yes, but I'd wait for https://github.com/GoogleChrome/chrome-launcher/pull/27 to get merged before doing any changes as that should enable running chrome from WSL :)

Reply:
> enable running chrome from WSL :)

Fixed by https://github.com/sonarwhal/sonar/commit/9be8747d4241e7b5907e61d05dba860a8f867498